lib/core/test/unit/Cardano/Wallet/DBSpec2.hs

```diff
@@ -797,17 +701,18 @@ instance ToExpr MWid where
   Top-level tests
 -------------------------------------------------------------------------------}
 
-spec :: Spec
+spec :: SpecWith (IO (DBLayer IO (SeqState DummyTarget) DummyTarget))
 spec = describe "DBLayer state machine tests" $ do
     it "Sequential tests" $
-        quickCheck (prop_sequential Nothing)
+        quickCheck . prop_sequential
```

Comment:
By the by @rvl, be careful with the use of `quickCheck`. quickCheck doesn't report errors with an error result or failure exit code. So, if the property fails, it just prints a counter example, but `hspec` doesn't report the test as fail. 
Instead, use `property` from QuickCheck and let hspec instrument the run of the property such that it can collects the result and properly report a failure if any!

Reply:
Fixed, thanks.